tests/system/test_openai_encoder.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from pinecone_text.dense import OpenAIEncoder
+from openai import OpenAIError
+
+
+DEFAULT_DIMENSION = 1536
+
+
+@pytest.fixture
+def openai_encoder():
+    return OpenAIEncoder()
+
+
+def test_init_without_openai_installed():
```

Comment:
I'm not sure I understand this test case. You're saying its "without_openai_installed", but then you expect OpenAI to be installed and importable?

Reply:
removed